server/resources/available.toggles

```diff
@@ -7,9 +7,9 @@
          "value": false
       },
       {
-         "key": "pipeline_config_single_page_app_key",
```

Comment:
I'm not sure anyone used it. If we're going to make it the default and we're still not sure, we should keep it. If not, it can go.